### DISCUSSION TITLE
The truth is just a doubt - mobs with complex objects

### DISCUSSION THREAD

**Discussion by mdmota** [mobxjs/mobx.dart#761] [General]
I have a large model with several properties, properties that are other objects and lists.
It is loaded via service after opening the application.
Some app features change part of this model, this part has to be reactive with observers.

The best option is to create a store that represents this complex model with the model's properties as observables?
And these properties don't need to be of type store?
how do i change the object data without having to put all its properties as observable?

Sample:
abstract class XControllerBase with Store {
  @observable
  ObservableList<Object> listX= ObservableList<Object>();

 @observable
  ObjectY yyyy;
}

**Comment by deadsoul44** [comment]
I hope metaprogramming will solve this pain. It does not make sense to add reactivity to data classes.
